code/modules/projectiles/projectile/special/plasma.dm

```diff
@@ -28,13 +28,25 @@
 	mine_range = 5
 
 /obj/item/projectile/plasma/adv/mech
-	damage = 10
+	damage = 25
 	range = 9
+	var/pressure_decrease_active = FALSE
+	var/pressure_decrease = 0.2
 	mine_range = 3
 
+
+/obj/item/projectile/plasma/adv/mech/Initialize()
+	. = ..()
+	if(!lavaland_equipment_pressure_check(get_turf(src)))

```

Comment:
Correct me if I'm wrong, but don't PKA projectiles calculate the pressure modifier based on the target square, not the source?

Reply:
KAs do it based on target, plasmacutters did it based on src.

Reply:
I think it would be better for it to be consistent, but I guess it's not a huge deal.

Reply:
The projectile itself isn't PASSGLASS so there aren't that many situations where the difference would affect things. And if you're using a mech for combat, just build an actual combat one instead of a glorified ranged toolbox.